src/irc/__init__.py

```diff
@@ -183,3 +188,8 @@ def _new_message(data, value, numerology_func=number_to_numerology):
         data += " " + app
 
     return data
+
+
+def chunks(l, n):
```

Comment:
Can you add typing to this, I think the following should work. You need to `from typing import List` too.

```suggestion
def chunks(message: str, n: int) -> List[str]:
```